dvc/command/imp_url.py

```diff
@@ -57,4 +60,10 @@ def add_parser(subparsers, parent_parser):
     import_parser.add_argument(
         "-f", "--file", help="Specify name of the DVC-file it generates."
     )
+    import_parser.add_argument(
+        "--remove-outs",
```

Comment:
not sure this us the best name for the option. It feels like w/o this option we are not removing outputs at all.

Reply:
true, @shcheklein , what about `--force` ? maybe `--no-ask-remove`? any suggestions?

Reply:
I'm actually fine with reusing this option, since we already have it. But we could also use `--force-remove-outs` instead.

Reply:
@shcheklein , does `--force-remove-outs` make sense to you?

Reply:
yep, I like it. I'm not sure I understand the compatibility with `dvc run` point